### PR TITLE
Update libs version

### PR DESCRIPTION
**Pull request by Vampouille**

### CODE REVIEW

**Comment by Vampouille** [comment]
@pvalsecc @sbrunner Can you review this PR ?

File: core/build.gradle [hunk 1]
```diff
@@ -176,14 +176,15 @@ dependencies {
...
-
-    compile ('org.apache.xmlgraphics:batik-transcoder:1.7'){
-        exclude module: 'fop'
-    }
-    compile ('org.apache.xmlgraphics:batik-codec:1.7')
+    compile ( 'org.apache.xmlgraphics:xmlgraphics-commons:2.2')
+    compile ('org.apache.xmlgraphics:batik-transcoder:1.8')
+//            {
+//                exclude module: 'fop'
+//            }
```
Comment (sbrunner) [inline core/build.gradle]: Shoulsn't we completely remove this exclude module?

File: examples/src/test/resources/examples/report/requestData.json [hunk 2]
```diff
@@ -10,49 +10,52 @@
...
+                    "type": "Feature",
+                    "geometry": {
+                        "type": "Point",
+                        "coordinates": [-8234500, 4982500]
+                    },
+                    "properties": {
+                        "type": "click"
+                    }
+                    }
+                    ]
```
Comment (sbrunner) [inline examples/src/test/resources/examples/report/requestData.json]: Pleas fix the indentation!

**Comment by pvalsecc** [comment]
Travis fails. apt cannot find some fonts.

File: .travis.yml [hunk 3]
```diff
@@ -2,6 +2,44 @@ language: java
...
+      - ttf-larabie-straight
+      - ttf-larabie-uncommon
+      - ttf-sjfonts
+      - ttf-staypuft
+      - ttf-summersby
+      - fonts-ubuntu-title
+      - ttf-xfree86-nonfree
+      - xfonts-intl-european
+      - xfonts-jmk
+      - xfonts-terminus
```
Comment (sbrunner) [inline .travis.yml]: Why do we needs all those fonts?

Comment (Vampouille) [inline .travis.yml]: On integration tests, there is some differences between travis and local execution. I think some fonts are missing on travis trusty instance. So I try to install almost all fonts to see if it changes something.
I will fix package list to fit trusty apt repo.

Comment (Vampouille) [inline .travis.yml]: Installing fonts don't fix image similarity so I revert this change.

**Comment by Vampouille** [comment]
Travis tests pass. @pvalsecc @sbrunner Are you agree to merge this PR ?

**Comment by sbrunner** [comment]
Many thanks :-)
